cmake/Packaging.cmake

```diff
@@ -94,6 +94,22 @@ endif()
 # cpack generation. CPACK_BINARY_DIR persists across cpack invocations
 set(LY_CMAKE_PACKAGE_DOWNLOAD_PATH ${CPACK_BINARY_DIR}/${CPACK_CMAKE_PACKAGE_FILE})
 
+# Scan the source and packages for licenses, then add the generated license results to the binary output folder
```

Comment:
this is also installing the 3p license file, can you add to the comment?

Reply:
Sure thing